packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -218,7 +217,6 @@ export class DetailsHeaderBase extends React.Component<IDetailsHeaderBaseProps,
         ref={this._onRootRef}
         onMouseMove={this._onRootMouseMove}
         data-automationid="DetailsHeader"
-        style={{ minWidth: rowWidth }}
         direction={FocusZoneDirection.horizontal}

```

Comment:
I think this will shrink the header to a smaller width than rows.

Reply:
it does not in my testing, because each column has a width set anyways
i've been testing with column resize as well
i'm honestly not sure why rowWidth exists in the first place if each field's width needs to be calculated individually anyways
@ThomasMichon any thoughts?